spec/job_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+require 'sequenceserver/job'
+require 'sequenceserver/blast/job'
+require 'rack/test'
+
+# Test Job class from module SequenceServer
+module SequenceServer
+    describe 'blank Job' do
+        my_job = Job.new
+
+        it 'should have an id' do
+            expect(my_job.id).to match(/\A(urn:uuid:)?[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}\z/i)
+        end
+
+        it 'should have a time of submission' do
+            expect(my_job.submitted_at).to be_kind_of(Time)
+        end
+
+        it 'should have an false exitstatus' do
+            expect(my_job.done?).to be_falsey
+        end
+
+        it 'should not be completed yet' do
+            expect(my_job.completed_at).to be(nil)
+        end
+
+        it 'should have a job directory' do
+            expect("#{__dir__}/dotdir/#{my_job.id}").not_to be_empty
+        end
+    end
+
+    describe 'BLAST Job' do
+
+        # Test Job class from module Blast and params
+        ENV['RACK_ENV'] = 'test'
+        include Rack::Test::Methods
+    
+        before do
+            SequenceServer.init(database_dir: "#{__dir__}/database/v5/sample")
+
+            @params = {
+                sequence: 'AGCTAGCTAGCT', 
```

Comment:
Can we have a more realistic example sequence that captures new lines and sequence identifiers?

```
>seq1
ATGCATGCATGCATGC
>seq2
ATGCATGCATGCATGC
```

Reply:
Sure!